src/pages/index.js

```diff
@@ -1,325 +1,253 @@
 import React from 'react'
-import { Container } from '@mui/material'
+import { Container, Grid } from '@mui/material'
```

Comment:
Used for the Resource section. Was not used for the cards simply because the Grid API for Material UI does not allow us to use the `a` element for Grid items and this impacts the shadow styling and hover effects implementation. As such, the cards are hand-rolled instead.